### PR TITLE
Cleanup of nearly all common functions

### PR DESCRIPTION
**Pull request by Winter259**
Mostly indentation & macros.

### CODE REVIEW

**Comment by Winter259** [comment]
You're not being anal if its on a list. #88

File: addons/common/functions/fnc_canGetInPosition.sqf [hunk 1]
```diff
@@ -44,218 +43,217 @@ _radius = 0;
...
 };
 
 // if you want into the cargo and you can't, then check ffv turrets aswell
 if (_position == "cargo") exitWith {
-  if (_return && {!_checkDistance || {_vehicle == vehicle _unit} || _fnc_isInRange}) then {true} else {
-    [_unit, _vehicle, "ffv", _checkDistance] call FUNC(canGetInPosition);
-  }
+    if (_return && {!_checkDistance || {_vehicle == vehicle _unit} || _fnc_isInRange}) then {true} else {
+        [_unit, _vehicle, "ffv", _checkDistance] call FUNC(canGetInPosition);
+    };
```
Comment (esteldunedain) [inline addons/common/functions/fnc_canGetInPosition.sqf]: I think the missing ; is optional in this context

File: addons/common/functions/fnc_closeDialogIfTargetMoves.sqf [hunk 2]
```diff
@@ -13,36 +13,35 @@
 #include "script_component.hpp"
```
Comment (esteldunedain) [inline addons/common/functions/fnc_closeDialogIfTargetMoves.sqf]: We need to scrap this function

Comment (Winter259) [inline addons/common/functions/fnc_closeDialogIfTargetMoves.sqf]: I can delete it in my PR if scrapping is in order. I leave the funeral arrangements to you?

File: addons/common/functions/fnc_getInPosition.sqf [hunk 3]
```diff
@@ -18,10 +18,10 @@
 #define CANGETINDRIVER      (isNull (driver _vehicle)             || {!alive driver _vehicle})               && {!lockedDriver _vehicle}           && {getNumber (_config >> "isUav") != 1}
 #define CANGETINTURRETINDEX (isNull (_vehicle turretUnit _turret) || {!alive (_vehicle turretUnit _turret)}) && {!(_vehicle lockedTurret _turret)} && {getNumber (_config >> "isUav") != 1}
 
-private ["_unit", "_vehicle", "_position", "_index"];
+private ["_position", "_index"];
```
Comment (esteldunedain) [inline addons/common/functions/fnc_getInPosition.sqf]: This function needs to be reviewed carefully. Github is being stupid about the diff

File: addons/common/functions/fnc_getTurrets.sqf [hunk 4]
```diff
@@ -24,26 +24,25 @@ _config = configFile >> "CfgVehicles" >> _type;
 
 _turrets = [];
 _fnc_addTurret = {
-  private ["_config", "_path", "_count", "_offset", "_index", "_path2", "_config2"];
+    private ["_count", "_offset", "_index", "_path2", "_config2"];
 
-  _config = _this select 0;
-  _path = _this select 1;
+    PARAMS_2(_config,_path);
```
Comment (esteldunedain) [inline addons/common/functions/fnc_getTurrets.sqf]: I guess params work here too

File: addons/common/functions/fnc_getWindDirection.sqf [hunk 5]
```diff
@@ -12,20 +12,20 @@
 #include "script_component.hpp"
```
Comment (esteldunedain) [inline addons/common/functions/fnc_getWindDirection.sqf]: This function could be scraped too

File: addons/common/functions/fnc_canGetInPosition.sqf (hunk 1 above)
Comment (jaynus) [inline addons/common/functions/fnc_canGetInPosition.sqf]: It is

**Comment by esteldunedain** [comment]
Thanks @Winter259! 

For the future, please make smaller PRs, and homogeneus if possible (e.g. on pr for tabulations, other for privates, etc). It's easier to review that way.

**Comment by Winter259** [comment]
Alright! :+1:

**Pull request merged**